website/templates/project/settings.mako

```diff
@@ -469,7 +469,8 @@
       window.contextVars.node.nodeType = ${ node['node_type'] | sjson, n };
       window.contextVars.nodeCategories = ${ categories | sjson, n };
       window.contextVars.wiki = window.contextVars.wiki || {};
-      window.contextVars.wiki.isEnabled = ${wiki.short_name in addons_enabled | sjson, n };
+      ##window.contextVars.wiki.isEnabled = ${wiki and wiki.short_name in addons_enabled | sjson, n };
```

Comment:
For whatever reason, without this change, I've never been able to get my local setup working. I don't know why the `wiki` is missing for me for any given project.  

```
AttributeError: 'Undefined' object has no attribute 'short_name'
```

Any suggestions on what I might be configuring incorrectly on my machine?